README.md

```diff
@@ -71,6 +71,8 @@ By default, the migration tool will load these values from a file named `vcs.jso
 Before initiating the migration process, first determine which command line arguments you wish to pass (if any).  The following arguments are currently supported:
 * `--vcs-file-path`: this flag allows you to pass a custom file path for your TFE_VCS_CONNECTION_MAP JSON file. If not specified, `vcs.json` will be used by default.
 * `--migrate-all-state`: this flag allows you to set the desired behavior for migrating state versions.  If passed, all versions of state will get migrated for all workspaces.  If not specified, only the current version of state for all workspaces will be migrated by default.
+* `--workspace-list-file-path`: this flag allows you to pass a custom file path for your SOURCE_WORKSPACE_LIST TXT file. If not specified, `workspace_list.txt` will be used by default.
```

Comment:
I would make this a `.json` file. Using `.txt` can be confusing.

Reply:
One thing to note about the issue with the `.txt` if we want to add any additional metadata to this workflow in the future, we'll have to change the format on people at that stage. Might be easier to just start with JSON, but you can also convince me otherwise.

Reply:
One last thing - we may want to call this the `migrate-select-json-path`, since eventually, users will want to use a single input file to select specific resources to migrate, not just workspaces. Might be another reason to use JSON.

Reply:
Good call.  Updated!